daemon/algod/api/algod.oas2.json

```diff
@@ -3972,6 +4043,103 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and benefit from group resource sharing. Also see property 'unnamed-resources' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing.",
+      "type": "object",
+      "required": [
+        "resources",
+        "max-cross-product-refs"
+      ],
+      "properties": {
+        "resources": {
+          "$ref": "#/definitions/SimulationUnnamedResourceAssignment"
+        },
+        "max-cross-product-refs": {
+          "description": "The maximum allowed number of asset holding and app local references.",
+          "type": "integer"
+        },
+        "asset-holdings": {
+          "description": "The unnamed asset holdings that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetHoldingReference"
+          }
+        },
+        "app-locals": {
+          "description": "The unnamed app local states that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationLocalReference"
+          }
+        }
+      }
+    },
+    "SimulationUnnamedResourceAssignment": {
+      "description": "This object contains a set of unnamed resources that were accessed during a simulation call.",
+      "type": "object",
+      "required": [
```

Comment:
I don't think I understand what any of these "max-*" values will contain.  I probably will eventually, when I read the code, but more description here could help.

Reply:
I think I under stand now, and if I do, I don't think we should return these numbers in the return json.  They seem to be a number you are calculating to help you avoid allowing too many refs.  I don't think that's very useful to report. Consider, especially, "max-accounts" which, confusingly, would start at 16, because it is the sum of potential accounts and potential foreignapps.  Getting back a 16 would imply then txn _could_ use 16 app references?

I'd rather see us just report when the txns touch, and not say anything about what they could have touched.

Reply:
I originally wanted to include these to show you some measure of why accessing an unnamed resource might fail. E.g. did you exceed the asset limit, or the total reference limit?

Though with the added weirdness of recent changes (like the max accounts behavior), the behavior of these limits has become more of an implementation quirk, so I think I'd be fine to remove them.

Reply:
Removed these fields in db2819027